lib/reporter.js

```diff
@@ -240,28 +242,20 @@ var generateReport = function (options) {
                                 embeddingType = embedding.media.type;
                             }
 
-                            if (embeddingType === 'text/plain' || embeddingType === 'text/html') {
+                            if (['text/plain', 'text/html', 'application/json'].includes(embeddingType)) {
 
                                 var decoded;
 
-                                if (embeddingType === 'text/html') {
-                                    decoded = new Buffer(embedding.data, 'base64').toString('ascii');
+                                if (isBase64(embedding.data)) {
```

Comment:
Are you sure it won't break the `text/html` embeddingTpe?

Reply:
sorry, I haven't seen any notification regarding this.

how do you mean in could break `text/html` embeddings?
if I remember correctly, cucumber-js will base64 encode html embeddings by default.